### ISSUE TITLE
Records not retrieved from store with namespaced model

### ISSUE THREAD

**Issue by juni0r** [funkensturm/ember-local-storage#71]
Hi,

when using a namespaced model (e.g. `blog/post`), records can be persisted but not retrieved from the store. This is because everything behind a '/' is interpreted as an id which fails for namespaced model names. 

The workaround we used is to change how the payload key and path name are derived from the model name in the serializer and adapter, respectively.

```js
// app/adapters/blog/post.js
import Adapter from 'ember-local-storage/adapters/adapter';

export default Adapter.extend({
  pathForType(modelName) {
    return this._super(modelName.replace('/', '::'));
  },
});
```

```js
// app/serializers/blog/post.js
import Serializer from 'ember-local-storage/serializers/serializer';

export default Serializer.extend({
  modelNameFromPayloadKey(key) {
    return this._super(key.replace('::', '/'));
  },

  payloadKeyFromModelName(modelName) {
    return this._super(modelName).replace('/', '::');
  },
});
```

**Comment by fsmanuel** [comment]
@juni0r Hey! Thanks for reporting. I'll look into it. I have never used namespaced models. Out of curiosity  have you used namespaced models with other adapters? Does it work?

**Comment by fsmanuel** [comment]
@juni0r I opened #73 that fixes the issue.

You need to set the `modelNamespace` on the perType Adapter:
```js
export default Adapter.extend({
  modelNamespace: 'blog'
}); 
```

Does this work for your use case?

**Comment by fsmanuel** [comment]
@juni0r can you review #73?

**Comment by fsmanuel** [comment]
@juni0r thanks for the review. I'll release later.

**Comment by fsmanuel** [comment]
@juni0r sorry for the delay. I released [1.1.0](https://github.com/funkensturm/ember-local-storage/releases/tag/v1.1.0)

**Comment by juni0r** [comment]
@fsmanuel No worries, I worked with master in the meantime. Thanks for releasing!
